mackerel-plugin-openldap/lib/openldap.go

```diff
@@ -83,23 +83,28 @@ func mergeStat(dst, src map[string]float64) {
 
 func latestCSN(sr *ldap.SearchResult) (time.Time, error) {
 	var res time.Time
-	for _, entry := range sr.Entries {
-		csns := make([]time.Time, len(entry.Attributes))
-		for i, attr := range entry.Attributes {
-			v := entry.GetAttributeValue(attr.Name)
```

Comment:
[memo]
https://godoc.org/gopkg.in/ldap.v2#Entry.GetAttributeValue

Reply:
forgot the test, oops!